packages/editor/src/client/global.less

```diff
@@ -93,3 +93,16 @@
     border-color: #1675e0;
   }
 }
+
+
+.authorLinks {
```

Comment:
Could we maybe add a class to the div that we try to modify? This creates a selector with a specificity level of 5 and is very hard to understand what is going to be affected.